model/commitqueue/commit_queue.go

```diff
@@ -29,6 +29,7 @@ func (m *Module) UnmarshalBSON(in []byte) error { return mgobson.Unmarshal(in, m
 
 type CommitQueueItem struct {
 	Issue           string    `bson:"issue"`
+	PatchId         string    `bson:"patch_id,omitempty"`
```

Comment:
Is the plan to maintain both PatchId and Version? Will it be possible to migrate to just using one of them, since now those IDs match, so they're now going to contain the same thing?